examples/fadion/classes.py

```diff
@@ -0,0 +1,49 @@
+class Classroom(object):
+    def __init__(self, count):
+        self.count = count
+        self.students = {}
+
+    def add_student(self, student):
+        if len(self.students) < self.count:
+            try:
+                if student.uid in self.students.keys():
+                    raise AlreadyInClassroomException("You can't add student with non unique id")
+            except AlreadyInClassroomException as e:
+                print("Exception! " + e.arg)
+            else:
+                self.students[student.uid] = student
+
+    def remove_student(self, student):
+        del self.students[student.uid]
```

Comment:
Sprawdziłem w dokumentacji i nawiasy nie są potrzebne przy del ponieważ jest to wyrażenie.
Tak samo jak do wersji 2.7 można było używać wyrażenia print, co w nowszych wersjach zostało zastąpione funkcją print()